.circleci/config.yml

```diff
@@ -1105,8 +1105,8 @@ workflows:
             branches:
               only:
                 - trunk
-                - release/*
-                - desktop/*
+                - /release\/.*/
+                - /desktop\/.*/
```

Comment:
Well spotted. Easy mistake to make, one would expect `release/*` to work out of the box, like in [other CI vendors](https://buildkite.com/docs/pipelines/branch-configuration#branch-pattern-examples).